src/getProjectId.js

```diff
@@ -21,10 +22,19 @@ module.exports = function(options, allowNull = false) {
 
     if (aliasCount === 0) {
       throw new FirebaseError(
-        "No project active. Run with " +
+        "No currently active project.\n" +
+          "To run this command, you have two options:\n" +
+          "- Run with " +
           clc.bold("--project <projectId>") +
```

Comment:
@bkendall had a way that he preferred this variable "project ID or project alias" to be referenced.  I _think_ this is what he suggested?

```suggestion
          clc.bold("--project <project-id | project-alias>") +
```

Reply:
See @bkendall 's comment below about what the help currently uses.  We should go with his suggestion in that comment below.

Reply:
Small nit: shouldn't it be --project=<project-id>?

Also, in firesite we say  --project=projectId_or_alias. Should we follow that?

Reply:
So, -P, --project, and --project= all work  identically - do we have a preference?

Reply:
@bkendall To make this call.